ac_qv_api.py

```diff
@@ -250,36 +263,42 @@ def draw_nets(self):
 				
 
 	def predict_qs(self, voltages):
-		# preproc the input
-		vg = vg.astype(np.float32)
-		vd = vd.astype(np.float32)
+		# requires voltage to be vg, vd
+		# both vg and vd must be numpy arrays
+
+		# preprocess the origin input and create adjoint input
 		if len(self.preproc_param) == 0:
 			self.preproc_param = pickle.load(
 				open(self.pickle_file_name, "rb" )
 			)
-		dummy_qs = np.zeros(len(vg))
-		preproc_data_arrays = preproc.ac_qv_preproc(
-			vg, vd, dummy_qs, 
+		dummy_qs = np.zeros(voltages[0].shape[0])
+		voltages, dummy_qs = preproc.ac_qv_preproc(
+			voltages, dummy_qs, 
 			self.preproc_param['scale'], 
 			self.preproc_param['vg_shift']
 		)
-		_preproc_data_arrays = [np.expand_dims(
-			x, axis=1) for x in preproc_data_arrays]
-		workspace.FeedBlob('DBInput_train/origin_input', _preproc_data_arrays[0])
-		workspace.FeedBlob('DBInput_train/adjoint_input', _preproc_data_arrays[1])
+		adjoint_input = np.ones((voltages[0].shape[0], 1))
+
+		# Expand dimensions of input and set data type of inputs
+		voltages = np.expand_dims(

```

Comment:
Can you clarify this in the comment (line 266) that voltages is an numpy array of size (batch size, input_dimention)? Thanks!